backend/src/environment.ts

```diff
@@ -67,6 +73,9 @@ export class Environment {
       this.deprecatedBinaries = PATHS.LINUX_DEPRECATED_BINARIES
     }
     this.manufacturerDetails = this.getManufacturerDetails()
+
+    this.oobAvailable = this.manufacturerDetails.product.platform === PLATFORM_CODES.REMOTEIT_PI
```

Comment:
This should be in the LAN class, where it can reference the env vars.

Reply:
moved